diff -ruN lupa-0.18-orig/setup.py lupa-0.18/setup.py
--- lupa-0.18-orig/setup.py	2011-01-22 19:32:34.090786000 +0200
+++ lupa-0.18/setup.py	2011-01-22 17:20:17.000000000 +0200
@@ -29,8 +29,8 @@
 basedir = os.path.abspath(os.path.dirname(__file__))
 
 def find_luajit_build():
-    static_libs = []
-    include_dirs = []
+    static_libs = ['/usr/lib/libluajit-5.1.a']
+    include_dirs = ['/usr/include/luajit-2.0']
 
     os_path = os.path
     for filename in os.listdir(basedir):
