diff -up mod_rpaf-0.6/Makefile.orig mod_rpaf-0.6/Makefile
--- mod_rpaf-0.6/Makefile.orig	2009-08-24 10:31:29.000000000 +0700
+++ mod_rpaf-0.6/Makefile	2009-08-24 10:32:35.000000000 +0700
@@ -1,7 +1,6 @@
 # Makefile for mod_rpaf.c (gmake)
 # $Id: Makefile 16 2007-12-13 03:40:22Z thomas $
-APXS=$(shell which apxs) 
-APXS2=$(shell which apxs2)
+APXS=/usr/sbin/apxs
 
 default:
 	@echo mod_rpaf:
@@ -41,7 +40,7 @@ mod_rpaf.so: mod_rpaf.c
 mod_rpaf.c:
 
 mod_rpaf-2.0.o: mod_rpaf-2.0.c
-	$(APXS2) -c -n $@ mod_rpaf-2.0.c
+	$(APXS) -c -n $@ mod_rpaf-2.0.c
 
 mod_rpaf-2.0.c:
 
@@ -49,7 +48,7 @@ install: mod_rpaf.so
 	$(APXS) -i -n mod_rpaf mod_rpaf.so
 
 install-2.0: mod_rpaf-2.0.o
-	$(APXS2) -i -n mod_rpaf-2.0.so mod_rpaf-2.0.la
+	$(APXS) -i -n mod_rpaf-2.0.so mod_rpaf-2.0.la
 
 clean:
 	rm -rf *~ *.o *.so *.lo *.la *.slo *.loT .libs/ 
